tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     AgentPool : 'Win-GPU-2019'
     ArtifactName: drop-nuget-dml
     JobName: 'Windows_CI_GPU_DML_Dev'
-    BuildCommand: --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --use_featurizers --enable_onnx_tests --use_telemetry  --use_winml --use_dml --cmake_generator "Visual Studio 16 2019"
+    BuildCommand: --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --use_featurizers --enable_onnx_tests --use_telemetry --use_dml --cmake_generator "Visual Studio 16 2019"
```

Comment:
use_winml?

Reply:
The lab has failing tests that need investigation so i have disabled gpu for now - once this bit change is in - i will be able to enable gpu in a more targetted fix.